setup.py

```diff
@@ -56,17 +56,17 @@ def readme():
     ],
     keywords='thumbor aws',
     install_requires=[
-        'python-dateutil',
+        'python-dateutil>=2.8',
         'thumbor>=7.0.0a2,<8',
         'aiobotocore==2.2.0',
+        'boto3>=1.9,<1.13',
     ],
     extras_require={
         'tests': [
-            'coverage',
-            'moto[server]',
-            'mock',
-            'pytest',
-            'pytest-asyncio'
+            'coverage>=6.5',
+            'moto[server]>=4.0',
+            'mock>=4.0',
+            'pytest>=7.2',
```

Comment:
You may be lacking pytest-asyncio. Same here if you can restrict the version range, it will improve the build time

Reply:
:+1: